src/api.js

```diff
@@ -159,7 +159,37 @@ api.declare({
   });
 });
 
-/** List namespaces inside another namespace */
+/** GET List namespaces inside another namespace */
+api.declare({
+  method:         'get',
+  route:          '/namespaces/:namespace?',
+  name:           'listNamespacesGet',
```

Comment:
This is what I did for expandScopes, where I was changing a GET to a POST.  You're going the opposite direction, changing a POST to a GET.  So the new method (this one) should be named `listNamespaces`, and the old method should be named `listNamespacesPost`.

Reply:
Changed the deprecated method's name to listNamespacesPost
https://github.com/taskcluster/taskcluster-index/pull/39/files#diff-e1d62d8c4965edf0e532dfa19da0891fR206